### PR TITLE
[#IR-150] Update the task summary tab

### PR DESCRIPTION
**Pull request by himanshu-garg-165**
Move retrospect and mark done column to the left.

Final columns to show under the task summary tab (in the order):

 - Pinned Columns - Done, Retrospect, ID, Summary

 - Scrollable Columns - Owner, Type, Status, Estimated Points, Total Hours, Sprint Points, Sprint Hours, Done At

### CODE REVIEW

File: src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.html [hunk 1]
```diff
@@ -31,7 +31,7 @@ <h6 class="subtitle">
         </div>
     </div>
     <mat-toolbar class="small-toolbar" color="primary">
-        <b>Members &nbsp;&nbsp;({{ getDisplayedRowCount() }})</b>
+        <b>Members&nbsp;&nbsp;({{ getDisplayedRowCount() }})</b>
```
Comment (chetanyakan) [inline src/app/retrospective/retrospect-task-modal/retrospect-task-modal.component.html]: Use a single space.

File: src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts [hunk 2]
```diff
@@ -12,28 +12,27 @@ export class ClickableButtonRendererComponent implements ICellRendererAngularCom
     label: string;
     isIcon = false;
     icon: string;
+    color: string;
     private params: any;
 
-    agInit(params: any): void {
-        this.params = params;
+    setRendererParams(params) {
```
Comment (chetanyakan) [inline src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts]: Change the name of this method to express its usage inside this component.
Also, make this method private.

File: src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts [hunk 3]
```diff
@@ -12,28 +12,27 @@ export class ClickableButtonRendererComponent implements ICellRendererAngularCom
...
-        let cellParams;
-        if (_.isFunction(cellRendererParams)) {
-            cellParams = cellRendererParams(params);
-        } else {
-            cellParams = cellRendererParams;
-        }
-        this.label = cellParams.label;
-        this.isIcon = cellParams.useIcon;
-        this.icon = cellParams.icon;
+        const cellParams = _.isFunction(cellRendererParams) ? cellRendererParams(params): cellRendererParams;
```
Comment (chetanyakan) [inline src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts]: Check if this is the correct way to use cellRendererParams.

Comment (himanshu-garg-165) [reply src/app/shared/ag-grid-renderers/clickable-button-renderer/clickable-button-renderer.component.ts]: We can provide a function for `cellRendererParams` while defining the column definitions of the ag-grid, it is better to do in this way, if we need to update the renderer params or we need to use the grid params to render a cell (using that renderer). We can also check if we can do the same in other places in the application.

**Pull request merged**